saleor/payment/gateways/braintree/__init__.py

```diff
@@ -33,23 +30,23 @@
 }
 
 
-def get_customer_data(payment: Payment) -> Dict:
+def get_customer_data(payment: Dict) -> Dict:
```

Comment:
Both `payment` and the output of this look like a good candidate for either a `dataclass` or a `namedtuple`? Notice how many hardcoded string constants are there in the following lines.

Reply:
`payment` being a custom type might be tricky, because then we'll need a separate package to just have the custom types. I believe the point is to have gateways separated from Saleor, but if we introduce a new type *in* Saleor (`payment` here is just a dict produced by `model_to_dict(Payment)`) to provide to gateways then they'll need that type also.